WrightTools/units.py

```diff
@@ -4,34 +4,31 @@
 # --- import --------------------------------------------------------------------------------------
 
 
-from __future__ import absolute_import, division, print_function, unicode_literals
+import collections
 
 import numpy as np
 import warnings
 
 
-# --- units ---------------------------------------------------------------------------------------
+# --- define --------------------------------------------------------------------------------------
 
 
 # units are stored in dictionaries of like kind. format:
 #     unit : to native, from native, units_symbol, units_label
 
 # angle units (native: rad)
-angle = {'kind': 'angle',
-         'rad': ['x', 'x', r'rad'],
+angle = {'rad': ['x', 'x', r'rad'],
          'deg': ['x/57.2958', '57.2958*x', r'deg']}
 
 # delay units (native: fs)
 fs_per_mm = 3336.
-delay = {'kind': 'delay',
-         'fs': ['x', 'x', r'fs'],
+delay = {'fs': ['x', 'x', r'fs'],
          'ps': ['x*1e3', 'x/1e3', r'ps'],
          'ns': ['x*1e6', 'x/1e6', r'ns'],
          'mm_delay': ['x*2*fs_per_mm', 'x/(2*fs_per_mm)', r'mm']}
 
 # energy units (native: nm)
-energy = {'kind': 'energy',
-          'nm': ['x', 'x', r'nm'],
+energy = {'nm': ['x', 'x', r'nm'],
```

Comment:
I'm mildly inclined to make the 'native' units for energy an actual energy unit (perhaps wn?)
Scaler multiples just seems cleaner to me (inverse still needed for nm, of course)

Is now the time to consider this change?

Reply:
sure, that's fine

Reply:
actually, let's skip it for now

I would accept a PR for it though :wink:

Reply:
I'll open an issue, mark as help wanted

Reply:
this is #449